### ISSUE TITLE
The offset is very picky

### ISSUE THREAD

**Issue by bluethekitsune** [fluddokt/opsu#195]
[Type your description here. Feel free to delete the info below if it's not relevant.]

---
**Version:** 0.15.0b
**OS:** Linux (armv8l)
**JRE:** The Android Project 0
**OpenGL version:** OpenGL ES 3.2 v1.r12p1-03dev0.228ab63cced004f840e7dd47b762a1d0 (ARM)
BOARD: universal8890
FINGERPRINT: samsung/hero2ltexx/hero2lte:7.0/NRD90M/G935FXXU1DQEB:user/release-keys
HOST: SWHE0716
MODEL: SM-G935F
INCREMENTAL: G935FXXU1DQEB
RELEASE: 7.0

I have observed a flaw in which some maps are mistimed while others work. It would be easier with some sort of wizard or calibration tool. Android ver. Otherwise the app is good

**Comment by fluddokt** [comment]
Yeah it would be nice though -200 usually works pretty ok for me.

**Comment by GreyMariGaming** [comment]
If i put the offset to -200 some of the beatmaps' timing was fixed but all of my other beatmaps' timing was broken.. I think the beatmaps have the right timing in osu! I think the beatmaps are broken

**Comment by fluddokt** [comment]
Added a poorly made calibration tool hopefully helps a bit.
Yea the offset is different for each song really can't figure out why.
